nats/aio/client.py

```diff
@@ -1474,13 +1474,20 @@ async def _process_msg(self, sid, subject, reply, data, headers) -> None:
                     sub._pending_size -= payload_size
 
                     await self._error_cb(
-                        SlowConsumerError(subject=subject, sid=sid, sub=sub)
+                        SlowConsumerError(
+                            subject=msg.subject,
+                            reply=msg.reply,
+                            sid=sid,
+                            sub=sub
+                        )
                     )
                     return
                 sub._pending_queue.put_nowait(msg)
             except asyncio.QueueFull:
                 await self._error_cb(
-                    SlowConsumerError(subject=subject, sid=sid, sub=sub)
+                    SlowConsumerError(
+                        subject=msg.subject, reply=msg.reply, sid=sid, sub=sub
```

Comment:
@wallyqs, `msg.subject` is of type `str`, but `subject` is of type `bytes`. That's what I meant by:

> SlowConsumerError.subject is of type bytes and this PR uses str

Reply:
Thanks I see now, it would be more useful as a `str` type this is better.